packages/ckeditor5-theme-lark/theme/ckeditor5-widget/widget.css

```diff
@@ -46,11 +46,19 @@
 	These styles show a different border for a blink of an eye, so `:focus` need to have same styles applied. */
 	&.ck-editor__nested-editable_focused,
 	&:focus {
-		@mixin ck-focus-ring;
 		@mixin ck-box-shadow var(--ck-inner-shadow);
 		@mixin ck-media-default-colors {
 			background-color: var(--ck-color-widget-editable-focus-background);
 		}
+
+		/**
+		 * Focus border should not be applied to table cells because it overrides the default table cell border color.
+		 * In other words - in some scenarios, the part of the table cell border is blue, which is not expected behavior
```

Comment:
I would not name the color directly as `blue` - better would be to call it `default editor focus color style` or something similar.